composer.json

```diff
@@ -35,32 +35,32 @@
         "ext-pdo": "*",
         "ext-zip": "*",
 
-        "assetic/framework": "~3.0",
-        "doctrine/dbal": "^2.6",
+        "assetic/framework": "^3.1",
+	"doctrine/dbal": "^3.0",
         "enshrined/svg-sanitize": "~0.16",
-        "laravel/framework": "^9.1",
-        "laravel/tinker": "^2.7",
+        "laravel/framework": "^11.0",
+        "laravel/tinker": "^2.8.1",
         "league/csv": "~9.1",
         "nesbot/carbon": "^2.0",
         "nikic/php-parser": "^4.10",
         "scssphp/scssphp": "~1.0",
-        "symfony/console": ">=6.0.9 <6.3.0",
-        "symfony/yaml": "^6.0",
+        "symfony/console": "^6.4|^7.0",
+        "symfony/yaml": "^6.4|^7.0",
         "twig/twig": "~3.0",
         "wikimedia/less.php": "~3.0",
         "wikimedia/minify": "~2.2",
         "winter/laravel-config-writer": "^1.0.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5.8",
```

Comment:
Why was this removed?

Reply:
Because it's included in Laravel's composer.json... do you want to keep this in storm ?

Reply:
I added it back in here.